### PR TITLE
[Accuracy diff No.116] Fix accuracy diff for paddle.nn.functional.ctc_loss API

### PR DESCRIPTION
**Pull request by ooooo-create**
显式传递 dim=-1，torch 的 log_softmax 具有默认参数 None，已经被标记废弃，并且和 paddle 默认的 -1 不一致
测试通过
图片

### CODE REVIEW

**Comment by cangtianhuang** [comment]
@cszdrg 帮忙review一下~
